src/main/groovy/com/morpheusdata/hyperv/HyperVProvisionProvider.groovy

```diff
@@ -223,7 +223,23 @@ class HyperVProvisionProvider extends AbstractProvisionProvider implements Workl
 	 */
 	@Override
 	ServiceResponse stopWorkload(Workload workload) {
-		return ServiceResponse.success()
+		def rtn = ServiceResponse.prepare()
+		try {
+			if (workload.server?.externalId) {
+				def hypervOpts = HypervOptsUtility.getAllHypervWorloadOpts(context, workload)
+				def results = apiService.stopServer(hypervOpts, hypervOpts.name)
+				if (results.success == true) {
+					rtn.success = true
+				}
+			} else {
+				rtn.success = true
+				rtn.msg = 'vm not found'
```

Comment:
This should be success false and a better message if we don't know the externalId.